metricbeat/docs/modules/googlecloud.asciidoc

```diff
@@ -83,6 +83,7 @@ metricbeat.modules:
 - module: googlecloud
   metricsets:
     - compute
```

Comment:
loadbalancing seems to be missing in the default config here?

Reply:
Added to default, thanks!